docker/irods_clients/manifest.json

```diff
@@ -9,6 +10,15 @@
         "baton-metaquery",
         "baton-put",
         "baton-specificquery",
+        "bcftools",
+        "bcftools",
```

Comment:
Duplicate line (`bcftools`)

Reply:
Ack - sorry - just fixed with @jmtcsngr's similar spot.

Reply:
fixed dups of `ienv`, `iinit` and `ils` too.